assets/sass/components/setup/_googlesitekit-adsense-connect-cta.scss

```diff
@@ -0,0 +1,186 @@
+/**
+ * AdSense Connect CTA styles.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-plugin {
+
+	.googlesitekit-setup__wrapper--adsense-connect {
+		$bp-nonMobile: $width-tablet + 1 + px;
```

Comment:
@techanvil Very nice. I can see usage of that breakpoint elsewhere. How about we move it to `assets/sass/config/_variables.scss`